Formula/gitleaks.rb

```diff
@@ -1,8 +1,9 @@
 class Gitleaks < Formula
   desc "Audit git repos for secrets"
   homepage "https://github.com/zricethezav/gitleaks"
-  url "https://github.com/zricethezav/gitleaks/archive/v1.24.0.tar.gz"
-  sha256 "6ba812be47976ca49bc2f5ab888c44ef41b824dd20fa9be5687f4ff6d185c2b1"
+  url "https://github.com/zricethezav/gitleaks/archive/v2.0.0.tar.gz"
+  sha256 "85a5c98dedeb4e85e07eb18247b63318aa266ef3046c2022eac949cc6f254da0"
+  head "https://github.com/zricethezav/gitleaks.git"
```

Comment:
Also remove head! You need to stop adding this everywhere. It should only be added if there is a good reason. E.g., upstream wants to do nightly testing with homebrew. Or you personally need and use it, and have tested it.

Reply:
yeah, sorry about the `head` issue spread, not quite understand the usage. I will remove it.